backon/src/backoff/mod.rs

```diff
@@ -12,3 +12,18 @@ pub use fibonacci::FibonacciBuilder;
 mod exponential;
 pub use exponential::ExponentialBackoff;
 pub use exponential::ExponentialBuilder;
+
+#[cfg(feature = "std")]
+fn f32() -> f32 {
```

Comment:
Thanks a lot for your PR! I suggest we split this PR into two:

- One is to make sure backon available on `std`
- The other one is to add `embassy_time` support.

Reply:
Yes, sure.